validation/gas_token_test.go

```diff
@@ -0,0 +1,97 @@
+package validation
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum-optimism/superchain-registry/validation/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetBytes_Success(t *testing.T) {
+	t.Parallel()
+	mockClient := &testutils.MockEthClient{
+		Responses: map[string][]byte{
+			string(testutils.MethodID("myMethod()")): {0xde, 0xad, 0xbe, 0xef},
+		},
```

Comment:
I wonder if we could have the mock expose a method for adding a mocked response, so that you could just do `mockClient.Prime("myMethod", {0xde, 0xad, 0xbe, 0xef})`.

Reply:
I'm going to refactor this whole `MockEthClient` following the example in the monorepo [here](https://github.com/ethereum-optimism/optimism/blob/develop/op-service/eth/transactions_test.go#L14-L29). I wrote the monorepo one 8 months ago but had forgotten where it lived 😆 .

That mock pattern eliminates the need for the custom `Responses` mapping referenced here